### ISSUE TITLE
tabBar.theme_barTintColor fades tabBar border

### ISSUE THREAD

**Issue by aarashgoodarzi** [wxxsw/SwiftTheme#148]
I'm using this line to change tabBar color :
`tabBar.theme_tintColor = ThemeColorPicker.pickerWithUIColors([.white, .black])`

when app loads with first theme there's no problem. 

<img width="380" alt="bordered" src="https://user-images.githubusercontent.com/39209402/110986653-29ca2480-8383-11eb-8e0a-88fdc6631777.png">

But if app loads with second theme when i change theme to the first, tabBar  border will be faded!

<img width="378" alt="no border" src="https://user-images.githubusercontent.com/39209402/110986682-351d5000-8383-11eb-9848-1bef71c2c1f9.png">

any help?

**Comment by fukemy** [comment]
hmm i face same problem here, for rounded border view, did u have any solutions?

**Comment by aarashgoodarzi** [comment]
Yes there is a solution. You have to give back the border by yourself, after theme changes. you can observe theme changes as mentioned in SwiftTheme documentation.

**Comment by aarashgoodarzi** [comment]
The best solution is to fix the issue by repo maintainers to avoid these hacky solutions :)

**Comment by fukemy** [comment]
hmm i know it, my situation is using round border for "view state", that mean when user online, u can see the green view, then it were inside tableview cell :(, as your guild, i have 2 case
  1. Reload table data -> the most simple case, but may lost current data, then screen will flashed when being reload
  2. Implement  notification for each cell :(, that's better, but will take more and more code (check when cell reused, or visible in tableview.....)
  
  So i think better if have simple. way to set theme for border

**Comment by aarashgoodarzi** [comment]
Store your data  then call reloadData() 
Implement notification for each cell is not a good practice

**Comment by wxxsw** [comment]
https://user-images.githubusercontent.com/921863/115112999-a7083b00-9fba-11eb-9ae8-85cf93dd8507.mov

Can you provide a demo, or provide steps to reproduce bug?
